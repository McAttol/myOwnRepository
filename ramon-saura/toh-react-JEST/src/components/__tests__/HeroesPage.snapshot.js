import renderer from "react-test-renderer";
import React from "react";
import HeroesPage from "../HeroesPage";
import { BrowserRouter as Router } from 'react-router-dom';


describe ('Hero Page snapshot', () => {
    const heroPageTree = renderer.create(<Router><HeroesPage/></Router>)

    it('shoul match', () => {
        expect(heroPageTree.toJSON()).toMatchSnapshot();
    })
})