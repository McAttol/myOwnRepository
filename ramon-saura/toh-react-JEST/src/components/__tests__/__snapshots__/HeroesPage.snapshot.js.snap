// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Page snapshot shoul match 1`] = `
Array [
  <p>
    <a
      href="/hero"
      onClick={[Function]}
    >
      + Add Hero
    </a>
  </p>,
  <ul />,
]
`;
